.gitignore

```diff
@@ -3,3 +3,12 @@
 /Cargo.lock
 
 *.~
+/.idea/.gitignore
```

Comment:
I think IDE specific ignores are better suited for your local `.git/info/exclude` or your global .gitignore (which you can set in your .gitconfig).